indico/web/templates/forms/protection_message_widget.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'forms/base_widget.html' %}
+
+{% block html -%}
+    {{ field.radio_widget(field, **input_args)  }}
+    {{ field.render_protection_message() | safe }}
+{%- endblock %}
+
+{% block javascript -%}
+    <script>
+        (function() {
+            'use strict';
+
+            $('#protection-form [name={{ field.name }}]').on('change', function() {
```

Comment:
I can't simply use `field.id` here since this is radio field. But I can use sth like this `[id^=protection_mode-]`